snooty/rstspec.toml

```diff
@@ -704,6 +704,8 @@ argument_type = ["path", "string", "uri"]
 options.uses-rst = "flag"
 options.uses-realm = "flag"
 options.preview = "flag"
+options.versions = "string"
```

Comment:
Should we remove this option if we're planning to keep the versions URL within the postprocessor?

Reply:
I was planning on supporting it unless we are truly striking it from the design. Are we ok with removing :versions: as an option for the openapi directive?

Reply:
I think it should be okay to remove for now if there's no current use case for it. I didn't see any code that suggests we're using the option's value (please correct me if I'm wrong!), so as-is, I don't think the option really does anything